pkg/loghttp/push/push.go

```diff
@@ -132,9 +133,15 @@ func ParseRequest(logger log.Logger, userID string, r *http.Request, tenantsRete
 		}
 		for _, e := range s.Entries {
 			totalEntries++
-			entriesSize += int64(len(e.Line))
-			bytesIngested.WithLabelValues(userID, retentionHours).Add(float64(int64(len(e.Line))))
-			bytesReceivedStats.Inc(int64(len(e.Line)))
+			var entryLabelsSize int64
+			for _, l := range e.NonIndexedLabels {
+				entryLabelsSize += int64(len(l.Name)) + int64(len(l.Value))
```

Comment:
you can do the conversion just once
```suggestion
				entryLabelsSize += int64(len(l.Name) + len(l.Value))
```